specification.markdown

```diff
@@ -778,3 +774,47 @@ minOccurs: 0
 type: Binary
 
 description: Used to void damaged data, to avoid unexpected behaviors when using damaged data. The content is discarded. Also used to reserve space in a sub-element for later use.
+
+# IANA Considerations
+
+This document creates a new IANA Registry called
+"CELLAR EBML Element ID Registry".
+
+Element IDs are described in section {{#element-id}}.  Element IDs are
```

Comment:
for style, `Element ID` and derivatives (plurals) are bickticked here. Also we usually don't force line breaks but let the make process add them in.